src/primitive.h

```diff
@@ -114,6 +114,7 @@ namespace toit {
   PRIMITIVE(add_entropy, 1)                  \
   PRIMITIVE(count_leading_zeros, 1)          \
   PRIMITIVE(popcount, 1)                     \
+  PRIMITIVE(compare_equal_vector, 2)         \
```

Comment:
I'd go ahead with the renaming we discussed. This isn't really a compare function and it doesn't return -1, 0, or 1. 

Something with vector equals something something.